iOS/third-party-podspecs/ComponentKit.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name = 'ComponentKit'
-  s.version = '0.21'
+  s.version = '0.22'
```

Comment:
I don't understand the change u did in `ComponentKit.podspec`. AFAIK if u don't mention anything like `:branch`, `:tag` in `s.source` it will pull in from master. So changing the version number from `0.21` to `0.22` will not serve any purpose. It will still pull from master.

Reply:
ComponentKit 0.22 tag is missing some commits that include enum values SonarKit depends on. The only way to get sonarkit to compile is by depending on ComponentKit latest commit(master). We could just define the dependency on the latest commit or open an issue in ComponentKit to update its 0.22 tag or potentially create a new tag.

Reply:
I did `pod install` with the current setup and for componentkit it pulled the code from current master. So all the required commits will be available. By setting `s.version = '0.21'` doesn't check out `0.21` tag since its not mentioned in the `s.source` argument. To checkout `0.21` tag, you will have to add `:tag` argument. Checkout [glog podspec](https://github.com/facebook/Sonar/blob/master/iOS/third-party-podspecs/glog.podspec). So tecchnically with current setup you can assign any value `s.version` still it will checkout master.

Reply:
Sorry I didn’t quite understand you. When you say current setup, you mean my pull request or master?

I am aware that with my setup, if you don’t define a tag in the s.source it will pull master. Setting a tag, either 0.21 or 0.22 will cause the sample app to not build.

Maybe I misunderstood you.

Reply:
> Sorry I didn’t quite understand you. When you say current setup, you mean my pull request or master?
By current setup I mean master.

> I am aware that with my setup, if you don’t define a tag in the s.source it will pull master. Setting a tag, either 0.21 or 0.22 will cause the sample app to not build.

True. So in order to not depend on master, componentkit guys have to release the new pod in Cocoapod. So in your current PR doing `s.version = '0.22'` still pulls master. So this change is not required, right?

Reply:
I think it would be good to get this out there in the meantime. I opened an issue in componentkit repo so they can address it. In the meantime we could get this merge so people playing with sonarkit and its sample app are able to build and run it.

As of today we are pulling componentkit from a local podspec in this repo. Ideally we should be pointing to a version in the podfile and rely on cocoapods master to do its job.

Reply:
>  In the meantime we could get this merge so people playing with sonarkit and its sample app are able to build and run it.

I can build sample app on master

Reply:
Oh wow. You can build master without any changes? I am unable to. I will double check when I get home.

Did you pod install? Pod update?

Reply:
I did pod install

Reply:
@priteshrnandgaonkar I just pulled master, pod repo update, pod install and I get:
`marcterns@Marcs-MBP ~/iOS/Sonar/iOS/Sample (master) $ pod install
Analyzing dependencies
Pre-downloading: `ComponentKit` from `https://github.com/facebook/ComponentKit.git`, commit `89c5e28e4ccc02927187a8aac6b7196f533cf86d`
Pre-downloading: `Folly` from `https://github.com/facebook/folly.git`, commit `1ed5d00ca4f43d5c81f4d731119e71509179dc99`
[!] Unable to find a specification for 'Folly'.`